test/webhook-proxy.test.ts

```diff
@@ -1,4 +1,5 @@
-import express, { Response } from "express";
+import express from "express";
+import type { Response } from "express";
```

Comment:
I would suggest putting these on the same line as before, there isn't a need for them to be on separate lines in this case
```suggestion
import express, { type Response } from "express";
```

Reply:
oh this is possible?

Reply:
Yup, you can do the same with exports as well
https://www.typescriptlang.org/docs/handbook/modules/reference.html#type-only-imports-and-exports

Reply:
ty, done ;)